tangelo/helpers/utils.py

```diff
@@ -73,8 +73,8 @@ def new_func(*args, **kwargs):
 
 
 # List all built-in backends supported
-all_backends = {"qulacs", "qiskit", "cirq", "braket", "projectq", "qdk", "pennylane", "sympy", "stim"}
-all_backends_simulator = {"qulacs", "qiskit", "cirq", "qdk"}
+all_backends = {"qulacs", "qiskit", "cirq", "braket", "projectq", "pennylane", "sympy", "stim"}
```

Comment:
Can we add `# "qdk"}` at the end of the line as an inline comment ?
Same with line 77.

This way we remember where it used to be if we get back to reactivating it.

Reply:
Sure!